### PR TITLE
Initial setup for PMM

### PR DESCRIPTION
**Pull request by brandonbk**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -345,3 +345,17 @@ services:
...
+    working_dir: /root/sites/plasticsmachinerymagazine
+    environment:
+      <<: *env
+      PORT: 80
+      EXPOSED_PORT: 12172
+      LIVERELOAD_PORT: 35172
+      GRAPHQL_URI: ${GRAPHQL_URI-https://graphql.base-cms.io/ebm/pmm}
+    ports:
+      - "12172:80"
+      - "35172:35172"
```
Comment (zarathustra323) [inline docker-compose.yml]: Set up your VSCode config to always add a new line at the end of files.
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: sites/plasticsmachinerymagazine/config/site.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  logo: {
+    src: 'https://cdn.plasticsmachinerymagazine.com/files/base/ebm/pmm/image/static/logo/site_logo.png',
```
Comment (zarathustra323) [inline sites/plasticsmachinerymagazine/config/site.js]: Update this to use imgix.

File: sites/plasticsmachinerymagazine/config/site.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    { alias: 'blow-molding', name: 'Blow Molding' },
+    { alias: 'thermoforming', name: 'Thermoforming' },
+    { alias: 'compounding', name: 'Compounding' },
+    { alias: 'injection-molding', name: 'Injection Molding' },
+    { alias: 'extrusion', name: 'Extrusion' },
+    { alias: 'recycling', name: 'Recycling' },
+    { alias: 'molds-tooling', name: 'Molds & Tooling' },
+    { alias: 'rotomolding', name: 'Rotomolding' },
+  ],
+  googleTagManagerID: 'GTM-KBM372Z',
```
Comment (zarathustra323) [inline sites/plasticsmachinerymagazine/config/site.js]: There's a lot of debate over whether `ID` within a camel-cased variable should be `ID` or `Id`. We tend to interchange it quite often. I'm okay with it if you are.

Comment (zarathustra323) [reply sites/plasticsmachinerymagazine/config/site.js]: Though, to be fair, we tend to use `Id` more than `ID`

File: sites/plasticsmachinerymagazine/config/site.js (hunk 2 above)
Comment (brandonbk) [reply sites/plasticsmachinerymagazine/config/site.js]: Done

File: sites/plasticsmachinerymagazine/config/site.js (hunk 3 above)
Comment (brandonbk) [reply sites/plasticsmachinerymagazine/config/site.js]: I think ID looks wrong, but...

**Pull request merged**